compose/prediction/Dockerfile-dev

```diff
@@ -3,13 +3,13 @@ ENV PYTHONUNBUFFERED 1
 RUN apt-get update && apt-get install -y tcl tk python3.6 python3.6-tk wget python-opencv
 RUN wget https://bootstrap.pypa.io/get-pip.py
 RUN python3.6 get-pip.py
-RUN ln -s /usr/bin/python3.6 /usr/local/bin/python
```

Comment:
This is the docker fix... I kept getting a file exists error

Reply:
Always good to document in the commit/code ;)